src/stretchy.js

```diff
@@ -143,6 +146,16 @@ const katexImagesData: {
     undergroup: [["leftgroupunder", "rightgroupunder"], 0.888, 342],
     xmapsto: [["leftmapsto", "rightarrow"], 1.5, 522],
     xtofrom: [["leftToFrom", "rightToFrom"], 1.75, 528],
+
+    // The next three arrows are from the mhchem package.
+    // In mhchem.sty, min-length is 2.0em. But these arrows might appear in the
+    // document as \xrightarrow or \xrightleftharpoons. Those have
+    // min-length = 1.75em, so we set min-length on these next three to match.
```

Comment:
I'm confused.  Wouldn't we want these commands to match `mhchem.sty`?

Reply:
The problem is that I'm trying to use one arrow to emulate both `mathtools` and `mhchem`. Those two packages both render a double-ended harpoon, and they are close, but not quite identical.  `mathtools` uses a larger arrow head. `mhchem` has a longer `min-length`.

My motivation has been to minimize the amount of code in `svgGeometry.js`. If you don't mind adding another dozen lines of code, then I could get a closer match to the original `mhchem`.

Reply:
These new arrows are only using new geometry in svgGeometry.js.  There doesn't seem to be any sharing of geometry between these new arrows and existing arrows.  Could we modify the new geometry that you already added?  What would need to be added?

Reply:
Two issues here:

1.  I need to retract part of what I said before. The `min-length` is independent of the `path` geometry in `svgGeometry.js`. So, yes, I can give all the `mhchem` reaction arrows the 2.0em `min-length` defined in the original `mhchem` package. To do that, I have to invent another four brand-new function names, then in the `katexImagesData` object, I can define the `min-length` as we want. That adds eight lines of code. But no new paths are necessary for min-length.

2.  `mhchem` has seven reaction arrows, not just the three new ones in this PR.  For the others, the proposed `mhchem` extension can make calls to extensible arrows that already exist. That's the code sharing I was referring to. And my stinginess with code means that `mhchem` will have to call the `mathtools` arrow `\xrightleftharpoons`, which has larger arrowheads than the original `mhchem` package.

To sum up: I can customize the `min-length` with another 8 line of code and I can customize the arrowhead size with a dozen lines more.

Reply:
I would be fine with any solution.
In Latex/mhchem (= mhchem.sty), I was very picky with the arrows. For instance, I made them all have the same length (when used without superscript or subscript). With MathJax/mhchem I couldn't be so picky. I used what I could get. There, the arrows are of different length, some even cannot stretch. So, I appreciate your perfectionism – and I could join in if you like (like the question if the min-length should be implemented inside the arrows themselves or be specified with their usage). On the other hand, the arrows are already much better than the ones of MathJax and the last grain of perfection could be added later on.